Makefile

```diff
@@ -40,6 +40,8 @@ NODE_VER := $(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - '"$(RELEASE_STREAM
 CTL_VER := $(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - '"$(RELEASE_STREAM)".[0].components.calicoctl.version')
 CNI_VER := $(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - '"$(RELEASE_STREAM)".[0].components.calico/cni.version')
 KUBE_CONTROLLERS_VER := $(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - '"$(RELEASE_STREAM)".[0].components.calico/kube-controllers.version')
+POD2DAEMON_VER := $(shell cat $(VERSIONS_FILE) | $(YAML_CMD) read - '"$(RELEASE_STREAM)".[0].components.flexvol.version')
```

Comment:
Do we need the flannel migration controller here as well?

Reply:
I had added that but removed it. My thinking was that it wasn't really a mainline installation (e.g. we don't include `flannel` for canal installations either. But I can add it in.

Reply:
I've added flannel-migration-controller 0fc51c12cdff32222c56b27e2307dccc1b85769e